### ISSUE TITLE
Plotting a range of truth values

### ISSUE THREAD

**Issue by albin-n** [dfm/corner.py#89]
I'm looking for a way to plot a range of truth values, sort of like a confidence interval. Ideally this would take the form of a semi-opaque wide line, just like the standard truths function, but the width of the line would represent the confidence width. 
Is there any way to do that? I have been playing around with the axes object, maybe that's a way forward. 
fig = corner.corner(data)
fig.axes[3].axhline(value, lw=2)

**Comment by dfm** [comment]
Yeah. That feature isn't currently supported so you'll have to reverse engineer the axis locations. You can probably do something like:

```
axes = np.reshape(fig.axes, (ndim, ndim))
```

To get the axis array in the right shape.

Hope that helps!
